contracts/utils/StringUtils.sol

```diff
@@ -0,0 +1,65 @@
+pragma solidity ^0.6.2;
+
+library StringUtils {
```

Comment:
This file is under the `utils` package, so suggest to remove the `utils` suffix. By the way, OZ v4.1.0 already has a library `Strings` under utils, please refer to the [code](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/Strings.sol).